### PR TITLE
Cleanup tempfiles

### PR DESCRIPTION
**Pull request by zhijianli88**

### CODE REVIEW

**Comment by justincormack** [comment]
can you rebase this?

**Comment by zhijianli88** [comment]
> can you rebase this?

Done

**Comment by zhijianli88** [comment]
ping

File: server/storage/sqlite_test.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ func sqlite3Setup(t *testing.T) (*SQLStorage, func()) {
 	tempBaseDir, err := ioutil.TempDir("", "notary-test-")
 	require.NoError(t, err)
 
-	dbStore := SetupSQLDB(t, "sqlite3", tempBaseDir+"test_db")
+	dbStore := SetupSQLDB(t, "sqlite3", tempBaseDir+"/test_db")
```
Comment (justincormack) [inline server/storage/sqlite_test.go:20]: can you use filepath.Join here rather than explicit "/".

Comment (zhijianli88) [reply server/storage/sqlite_test.go]: make sense, Done

**Review by justincormack — APPROVED** [review]

**Comment by justincormack** [comment]
can you rebase this please? should fix the codecov issue I believe.

**Comment by zhijianli88** [comment]
i think i had already rebased to the latest  version, there is no new merges to master for a month

**Comment by zhijianli88** [comment]
since the CI said the 2nd and 2rd commits failed, try push only 1st commit first.

**Comment by zhijianli88** [comment]
> i think i had already rebased to the latest version, there is no new merges to master for a month

it still didn't work

**Comment by justincormack** [comment]
Ignoring codecov error as incorrect.

**Pull request merged**